miden-lib/asm/kernels/transaction/lib/prologue.masm

```diff
@@ -352,30 +251,28 @@ proc.validate_new_account
 
         if.true
             # assert the fungible faucet reserved slot is initialized correctly (EMPTY_WORD)
-            or or or assertz.err=ERR_PROLOGUE_NEW_FUNGIBLE_FAUCET_NON_EMPTY_RESERVED_SLOT
+            padw eqw assert.err=ERR_PROLOGUE_NEW_FUNGIBLE_FAUCET_NON_EMPTY_RESERVED_SLOT
```

Comment:
I'd keep the previous approach: `padw eqw` takes 19 cycles while `or or or` takes just 3.